### PR TITLE
Alphonzo79 docdb collection initialization

### PR DESCRIPTION
**Pull request by keithcraig**
Update REST API version for DocDB and work around possible versioning conflicts with other databases and collections in the same DocDB instance.

### CODE REVIEW

File: Common/Utility/DocDbRestUtility.cs [hunk 1]
```diff
@@ -215,31 +173,60 @@ public DocDbRestUtility(IConfigurationProvider configProvider)
...
+                result.ContinuationToken = responseHeaders["x-ms-continuation"];
+
+                return result;
+            }
+        }
+
+        private async Task<string> QueryDocDbInternal(WebClient preparedWebClient, string endpoint, string queryString, Dictionary<string, Object> queryParams, 
+            string resourceType, string resourceId, int pageSize = -1, string continuationToken = null)
+        {
+            preparedWebClient.Headers.Set("Content-Type", "application/query+json");
```
Comment (jamesweb-ms) [inline Common/Utility/DocDbRestUtility.cs]: No null check on preparedWebClient

Comment (keithcraig) [inline Common/Utility/DocDbRestUtility.cs]: Will fix

Comment (jamesweb-ms) [inline Common/Utility/DocDbRestUtility.cs]: Actually why are you passing in WebClient...  I don't see anything being "prepared" just a using wrapper around this function.  Its cleaner if this function creates and disposes of its own client.

Comment (keithcraig) [inline Common/Utility/DocDbRestUtility.cs]: Check out line 193 of DocDbRestUtility.cs--in that case we query the `ResponseHeaders` property on the client *after* the call. (If we `.Dispose()` the client inside the method, we would need to extract the `ResponseHeaders` inside the method and return them in addition to the existing return value--possible but a bit messy.)

Thoughts?

Comment (jamesweb-ms) [inline Common/Utility/DocDbRestUtility.cs]: You are relying on caller to dispose, which is bad practice if can be avoided.  I'd either go out route, or create a class that can return both your string and response header.  Not sure if other callers might need document count, but I can see that being useful.

**Comment by keithcraig** [comment]
We're implementing the feedback and should have updated code soon.

**Comment by keithcraig** [comment]
Updated and ready for review!

**Pull request merged**